consensus/polybft/consensus_runtime.go

```diff
@@ -509,11 +531,6 @@ func (c *consensusRuntime) calculateUptime(currentBlock *types.Header, epoch *ep
 	return commitEpoch, nil
 }
 
-// InsertExitEvents is an implementation of checkpointBackend interface
-func (c *consensusRuntime) InsertExitEvents(exitEvents []*ExitEvent) error {
-	return c.state.insertExitEvents(exitEvents)
-}
-
 // BuildEventRoot is an implementation of checkpointBackend interface
 func (c *consensusRuntime) BuildEventRoot(epoch uint64) (types.Hash, error) {
```

Comment:
Why this isn't part of `checkpointManager`?

Reply:
It will be. There are two more tasks for `checkpointManager`. Didn't want to have all the changes in one big PR, so I will break them into smaller PRs.